msamp/common/dtype/floating.py

```diff
@@ -15,24 +15,29 @@ class Floating:
     qfp_max: dict = {}
 
     @staticmethod
-    def _get_fp_max(exp, man, inf_existed=True):
+    def _get_fp_max(exp, man, inf_existed=True, nan_existed=True):
```

Comment:
Seems we don't need to revise this file, because parameter `nan_existed` is never set to `False` in new code